acceptance/framework/config/config.go

```diff
@@ -217,21 +216,8 @@ func (t *TestConfig) entImage() (string, error) {
 	}
 
 	// Otherwise, assume that we have an image tag with a version in it.
-	consulImageSplits := strings.Split(v.Global.Image, ":")
-	if len(consulImageSplits) != 2 {
-		return "", fmt.Errorf("could not determine consul version from global.image: %s", v.Global.Image)
-	}
-	consulImageVersion := consulImageSplits[1]
-
-	var preRelease string
-	// Handle versions like 1.9.0-rc1.
-	if strings.Contains(consulImageVersion, "-") {
-		split := strings.Split(consulImageVersion, "-")
-		consulImageVersion = split[0]
-		preRelease = fmt.Sprintf("-%s", split[1])
-	}
-
-	return fmt.Sprintf("hashicorp/consul-enterprise:%s%s-ent", consulImageVersion, preRelease), nil
+	// Use the same Docker repository and tagging scheme, but replace 'consul' with 'consul-enterprise'.
+	return strings.Replace(v.Global.Image, "/consul:", "/consul-enterprise:", 1), nil
```

Comment:
Will this work? Enterprise images have the redundant `-ent` at the end as well for release versions

eg. `hashicorp/consul-enterprise:1.16-ent` https://hub.docker.com/r/hashicorp/consul-enterprise/tags

Reply:
hmm it probably works with the [nightly version](https://hub.docker.com/r/hashicorppreview/consul-enterprise/tags) since that doesn't include the `-ent` at the end of the tag. We use the nightly version in our in-development branches but we do pin the Consul version when preparing for release. We should verify this doesn't break in that case.

Reply:
As an aside, I'd argue the Consul team should get rid of the `-ent` at the end of the tag in Consul Enterprise, I wonder why it's even still there

Reply:
Ah right, I forgot this was inconsistent w/ the nightly tagging and we don't yet drop it in the release tags 🤦🏻‍♂️  great catch.

Agree, we should change the release tagging scheme to drop `-ent`. That came up recently in Slack; if I can find some relevant updates I'll post them here.

Reply:
@wilkermichael updated, LMK if the new version looks good to you. I preferred assuming there would not be an `-ent` suffix except in the case of a prod release since local image tagging should also act "normally".

(IIRC the `-ent` suffix comes from normalization of the [product version](https://github.com/hashicorp/bob/blob/12ad5978ca0a6f4e0f222f962732b7a432b4037b/internal/commands/triggerpromotion.go#L241) we pass to `bob`; the suffix gets converted from `+ent` to `-ent` between Artifactory and CRT's promotion scripts, from what I can see implicitly.)